libs/remix-ui/workspace/src/lib/actions/workspace.ts

```diff
@@ -150,15 +149,21 @@ export const loadWorkspacePreset = async (template: 'gist-template' | 'code-temp
       }
       break
 
-    case 'default-template':
-      // creates a new workspace and populates it with default project template.
-      // insert example contracts
-      for (const file in examples) {
-        try {
-          await workspaceProvider.set(examples[file].name, examples[file].content)
-        } catch (error) {
-          console.error(error)
+    default:
+      try {
```

Comment:
if this somehow fails we should now probably use `displayNotification`

Reply:
added